### PR TITLE
Adding RangeSchedule class to docs

### PR DESCRIPTION
**Pull request by danielstocker**
- Added new doc for RangeSchedule with description of constructor
- Linked to solver inputs for usage details

### CODE REVIEW

**Comment by dime10** [comment]
Thanks for adding this!

Closes #234

File: articles/optimization-rangeschedule.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+```py
+from azure.quantum.optimization import RangeSchedule
+```
+
+Some solvers allow you to specify an instance of the RangeSchedule class as an input. 
+A RangeSchedule can be created with two different types of schedules. (either linear or geometric)
+We recommend reviewing our [Solver Inputs](/azure/quantum/optimization-input-formats) documentation for more details.
+
+
```
Comment (dime10) [inline articles/optimization-rangeschedule.md:25]:
```suggestion
```

File: articles/optimization-rangeschedule.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+### Constructor
+
+- `schedule_type`: This is a string indicating the type of schedule. The accepted values are `linear` and `geometric`.
+- `initial`: A floating point value indicating the initial value of the schedule.
+- `final`: A floating point value indicating the final value of the schedule.
+
+ For instance, if you want to create a linear range schedule that increases from 0 to 5 over time:
+ 
+ ```py
+ RangeSchedule("linear", 0, 5)
```
Comment (dime10) [inline articles/optimization-rangeschedule.md:35]:
```suggestion
For instance, if you want to create a linear range schedule that increases from 0 to 5 over time:
 
```py
RangeSchedule("linear", 0, 5)
```

File: articles/optimization-rangeschedule.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+ Or, if you want to create a geometric range schedule that decreases from 1 to 0:
+  ```py
+ RangeSchedule("geometric", 1, 0)
```
Comment (dime10) [inline articles/optimization-rangeschedule.md:40]:
```suggestion
Or, if you want to create a geometric range schedule that decreases from 1 to 0:

```py
RangeSchedule("geometric", 1, 0)
```

File: articles/optimization-rangeschedule.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+# Quantum optimization range schedule
+
+## RangeSchedule
+
+```py
+from azure.quantum.optimization import RangeSchedule
+```
+
+Some solvers allow you to specify an instance of the RangeSchedule class as an input. 
+A RangeSchedule can be created with two different types of schedules. (either linear or geometric)
```
Comment (dime10) [inline articles/optimization-rangeschedule.md:22]:
```suggestion
A RangeSchedule can be created with two different types of schedules (either linear or geometric).
```

**Review by dime10 — CHANGES_REQUESTED** [review]
Would like someone specific to look over it content-wise?
Otherwise it looks good to me.

**Comment by danielstocker** [comment]
@dime10 I think it's fine to merge as is. I've taken all information from the source code directly. @frtibble was kind enough to send me a link :)

**Review by frtibble — APPROVED** [review]

**Review by dime10 — APPROVED** [review]

**Pull request merged**